spec/shared/system/mappable.rb

```diff
@@ -291,6 +291,16 @@
       expect(page).to have_css(".map_location")
     end
 
+    scenario "Should display marker on #{mappable_factory_name} show page", :js do
+      arguments[:id] = mappable.id
+
+      visit send(mappable_show_path, arguments)
+
```

Comment:
Since the setup for this test is exactly the same as the previous test, and system tests are quite slow, what do you think about merging this test with the previous one? Something like:

```ruby
scenario "Displays map and marker on #{mappable_factory_name} show page", :js do
  arguments[:id] = mappable.id

  visit send(mappable_show_path, arguments)

  expect(page).to have_css(".map_location")

  within ".map_location" do
    expect(page).to have_css(".map-icon")
  end
end
```

Reply:
Considering that `mappable` show pages only show the map when a marker is defined it has perfect sense to me. Thanks